lts2nix/Main.hs

```diff
@@ -76,3 +78,5 @@ lts2plan compilerPackagesMap lts = Plan { packages, compilerVersion, compilerPac
             _               -> Nothing
         , packageFlags = Map.lookupDefault Map.empty name flags
         })
+
+    packages = packages' `Map.union` compilerPackages'
```

Comment:
I think we want the union the other way around. If packages provides overrides for the compilerPackages, we want them to override those, not rely on the compilerPackages.

Reply:
This was my idea too. According to the [docs](http://hackage.haskell.org/package/unordered-containers-0.2.10.0/docs/Data-HashMap-Strict.html#v:union):

> The union of two maps. If a key occurs in both maps, the mapping from the first will be the mapping in the result.

so `packages'` has preference over `compilerPackages'`.

Reply:
Ohh... that's somewhat counter intuitive... thanks!